Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/ProjectController.cs

```diff
@@ -60,19 +64,23 @@ public ProjectController(PromactOauthDbContext appContext, IProjectRepository pr
             {
                 var user = await _userManager.FindByNameAsync(User.Identity.Name);
                 var userRole = await _userManager.IsInRoleAsync(user, "Employee");
+                _logger.LogInformation("UserRole Employee  "+userRole);
                 if (userRole == true)
                 {
+                    _logger.LogInformation("call project repository for User");
                     return await _projectRepository.GetAllProjectForUser(user.Id);
                 }
                 else
                 {
+                    _logger.LogInformation("call project repository for projects");
                     return await _projectRepository.GetAllProjects();
                 }
             }
             catch (Exception ex)
```

Comment:
I think I have conveyed this that remove all try.. catch unless you are catching any specific exception